transforms/unchain-variables.js

```diff
@@ -0,0 +1,35 @@
+/**
+ * Unchains chained variable declarations.
+ */
+module.exports = function(file, api) {
+  const jscodeshift = api.jscodeshift;
```

Comment:
I'm not a huge fan of using `jscodeshift` over `j` because of how often it needs to be typed. Wish JavaScript had something like a `with` statement :P